bedrock/mozorg/templates/mozorg/about/leadership.html

```diff
@@ -159,7 +159,7 @@ <h3 id="mgmt-corp"><a href="{{ url('foundation.moco') }}">{{ _('Mozilla Corporat
           <img class="photo" itemprop="image" alt="" src="{{ static('img/mozorg/about/leadership/mark-mayo.jpg') }}">
           <span class="fn" itemprop="name">Mark Mayo</span>
         </a>
-        <span class="title" itemprop="jobTitle">{{ _('VP and General Manager, Firefox') }}</span>
+        <span class="title" itemprop="jobTitle">{{ _('Senior Vice President, Firefox') }}</span>
 
```

Comment:
&lt;span class="note"&gt;{{ _('Steering Committee') }}&lt;/span&gt; here?

Should this be moved up to group with other SC members?